app/serializers/activitypub/note_serializer.rb

```diff
@@ -86,6 +86,10 @@ def local?
     object.account.local?
   end
 
+  def license
```

Comment:
Use `key` option of `attribute` method. See http://www.rubydoc.info/gems/active_model_serializers/0.10.6/ActiveModel/Serializer#attribute-class_method for details.